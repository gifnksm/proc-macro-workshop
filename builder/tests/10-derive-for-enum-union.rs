use derive_builder::Builder;

#[derive(Builder)]
pub enum Enum {}

#[derive(Builder)]
pub union Union {
    dummy: i32,
}

fn main() {}
