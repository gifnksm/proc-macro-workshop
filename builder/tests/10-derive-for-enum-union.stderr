error: cannot derive `Builder` for enum `Enum`
 --> $DIR/10-derive-for-enum-union.rs:4:1
  |
4 | pub enum Enum {}
  | ^^^^^^^^^^^^^^^^

error: cannot derive `Builder` for union `Union`
 --> $DIR/10-derive-for-enum-union.rs:7:1
  |
7 | / pub union Union {
8 | |     dummy: i32,
9 | | }
  | |_^
